src/System.Security.Cryptography.Encoding/src/Resources/Strings.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Cryptography_Cms_Invalid_Encoding" xml:space="preserve">
```

Comment:
A couple of thoughts about this error string:
* The string you used maps to winerror.h CRYPT_E_ASN1_EOD.
* It looks like mostly Windows will report CRYPT_E_ASN1_CORRUPT if it doesn't think the value is a UTCTime
* That string is "ASN1 corrupted data."
* Now that I've had all these thoughts, I have to ask why the error message from the DER reader says "CMS" in it.  "Der" makes a lot more sense.